commcare_export/commcare_hq_client.py

```diff
@@ -141,15 +141,16 @@ def iterate_resource(resource=resource, params=params):
                     else:
                         more_to_fetch = False
 
-                self.checkpoint(paginator, batch)
+                    self.checkpoint(paginator, batch)
                 
         return RepeatableIterator(iterate_resource)
 
     def checkpoint(self, paginator, batch):
         from commcare_export.commcare_minilinq import DatePaginator
         if self._checkpoint_manager and isinstance(paginator, DatePaginator):
             since_date = paginator.get_since_date(batch)
-            self._checkpoint_manager.set_batch_checkpoint(checkpoint_time=since_date)
+            if since_date:
```

Comment:
can you log a warning here if this is `None`

Reply:
added - feel free to suggest text changes